addons/event/static/src/client_action/event_registration_summary_dialog.js

```diff
@@ -46,16 +46,22 @@ export class EventRegistrationSummaryDialog extends Component {
             this.printSettings.iotPrinterId = this.registration.iot_printers[0].id;
         }
 
+        this.willAutoPrint = 
+            this.registration.status === 'confirmed_registration' &&
+            this.printSettings.autoPrint && this.useIotPrinter &&
+            this.hasSelectedPrinter() && !this.registration.has_to_pay;
```

Comment:
has_to_pay is defined only in event_sale

Reply:
I think this is fine - in the case it's not installed, `has_to_pay` will be `undefined` which will still evaluate as `false`. The XML template also already uses it prior this PR.

Reply:
Well the fact it is wrong before this pr does not allow to continue bieng wrong ...